test/unit/limiter_test.rb

```
@@ -162,6 +162,61 @@ def test_fingerprint_is_sha256_hex
+      unlimited: Wurk::Limiter.unlimited
+    }
+    limiters.each do |type, l|
+      s = l.status
+
+      %i[used limit reset_at available?].each { |k| assert s.key?(k), "#{type} status missing #{k}" }
+    end
+  end
+
+  def test_bucket_status_tracks_usage_and_reset_boundary # rubocop:disable Minitest/MultipleAssertions
+    l = Wurk::Limiter.bucket("sbu-#{@suffix}", 3, :minute, wait_timeout: 0)
+    2.times { l.within_limit {} }
```

RuboCop (1.86.2)

[warning] Empty block detected.

(Lint/EmptyBlock)